frontend/src/_components/MultiSelectUser.jsx

```diff
@@ -50,11 +50,11 @@ function MultiSelectUser({
             // eslint-disable-next-line no-unused-vars
             onClick={selectOption}
```

Comment:
Thank you for your feedback! I have thoroughly tested the code and made the necessary changes. I have removed the redundant function that was not required anymore since it was not being repeated. 

Initially, I hesitated to implement it as you suggested because I was concerned that the checkbox input field would lose its purpose. But, upon further consideration, I realized that it may serve as a decorator. All in all, thank you for the suggestion! Please let me know if there were any unnecessary changes I made.